recipe/thirdparty/recipe.go

```diff
@@ -178,6 +184,30 @@ func (r *Recipe) getEmailForUserId(userID string, userContext supertokens.UserCo
 	}, nil
 }
 
+func (r *Recipe) getTenantIdForUserId(userID string, userContext supertokens.UserContext) (multitenancymodels.TenantIdResult, error) {
+	userInfo, err := (*r.RecipeImpl.GetUserByID)(userID, userContext)
+	if err != nil {
+		return multitenancymodels.TenantIdResult{}, err
+	}
+
+	if userInfo == nil {
+		return multitenancymodels.TenantIdResult{
+			UnknownUserIDError: &struct{}{},
+		}, nil
+	}
+
+	tenantId := tpmodels.DefaultTenantId
```

Comment:
also, move the defaultTenantId to the multitenancy recipe?

Reply:
Done